ai/smart-labeling/how-to/delete-labeling-workspace.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: How to delete a labeling workspace
+description: This page explains how to delete a labeling workspace
+dates:
+  validation: 2021-11-05
+  posted: 2021-11-05
+---
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  
+- You have a [Smart Labeling](/ai/smart-labeling/quickstart) workspace
+
+</Message>
+
+1. Click **Smart Labeling** in the **AI** section of the side menu. The list of your workspaces displays.
+
+2. Click on the workspace you want to access. The workspace overview displays.
+
+3. Scroll down to the **Delete your labeling workspace** section. 
+
+4. Click **Delete** and type **DELETE** in the pop-up. Then click **Delete this labeling workspace** to permanently delete your workspace.
+
+<Message typ="note">
+
+Deleting the labeling workspace does **not** delete the associated S3 buckets. To avoid incurring Object Storage costs, it is recommended to delete the buckets separately if you no longer need the data.
```

Comment:
Also add warning that once they delete the workspace, all their annotation data is deleted so they have to make sure to have saved/downloaded/backed it up first. You could copy/paste the wording in the console wording.